omero/sysadmins/unix/server-install-homebrew.txt

```diff
@@ -248,14 +248,15 @@ Start the PostgreSQL server.
     $ brew services start postgresql
     $ pg_ctl -D /usr/local/var/postgres/ -l /usr/local/var/postgres/server.log start
 
-Create a user, a database and add the PL/pgSQL language to your database.
+Create a user, a database and, if using PostgreSQL 8.4, add the PL/pgSQL
+language to your database.
 
 ::
 
     $ createuser -P -D -R -S db_user
     Enter password for new role:       # db_password
     Enter it again:       # db_password
-    $ createdb -O db_user omero_database
+    $ createdb -E UTF8 -O db_user omero_database
```

Comment:
Could you also open a PR to modify `OMERO-homebrew-install.sh` to include this option and have it tested by the daily job?